tests/test_models.py

```diff
@@ -8,4 +8,4 @@
 def test_with_device_categories(admin_client):
     factories.Device.create_batch(10, provider=factories.Provider())
     provider = models.Provider.objects.with_device_categories().get()
-    assert provider.device_categories == {"bicycle": 4, "scooter": 3, "car": 3}
+    assert sum(provider.device_categories.values()) == 10
```

Comment:
Interesting, the test was flaky? Leave it unchanged, I'll improve it after you merge.

Reply:
ok

Reply:
The number of affected devices for each category can change between test run